code/game/gamemodes/cm_initialize.dm

```diff
@@ -68,7 +72,7 @@ Additional game mode variables.
 	var/monkey_amount		= 0 //How many monkeys do we spawn on this map ?
 	var/list/monkey_types	= list() //What type of monkeys do we spawn
 	var/latejoin_tally		= 0 //How many people latejoined Marines
-	var/latejoin_larva_drop = 7 //A larva will spawn in once the tally reaches this level. If set to 0, no latejoin larva drop
+	var/latejoin_larva_drop = LATEJOIN_LARVA_REQUIRED_NUM
```

Comment:
sorry to be clear remove the variable latejoin_larva_drop and replace all references with that define

Reply:
Had to do it like this.